DEFRA/waste-permits
# We 2466 implement simple pdf file render

This is the first half of the work for allowing a copy of the application to be rendered to PDF and sent to sharepoint. This does not expose any new functionality to the user, that comes later.

To test locally:

Try this saved aplication:
http://localhost:8000/r/1b917ab7412b4ca6904aa10f2009f62a

Then I've added a route to download the dpf (not available via UI, yet):
http://localhost:8000/check-before-sending/pdf-download

Things to note:
- I've added the `pdfmake` lib (this is really a browser lib so...)
- I've had to add a `.labrc.js` to get around the global var that babel adds

Hope that all makes sense!?
